Doc/data/refcounts.dat

```diff
@@ -1399,6 +1627,11 @@ PyMethod_New:PyObject*:class:0:
 PyMethod_Self:PyObject*::0:
 PyMethod_Self:PyObject*:im:0:
 
+PyModule_Add:int:::
+PyModule_Add:PyObject *:module:0:
+PyModule_Add:const char *:name:0:
+PyModule_Add:PyObject *:value:0:stolen
```

Comment:
There is a comment at the top of the file that says:

> # XXX NOTE: the 0/+1/-1 refcount information for arguments is
# confusing!  Much more useful would be to indicate whether the
# function "steals" a reference to the argument or not.  Take for
# example PyList_SetItem(list, i, item).  This lists as a 0 change for
# both the list and the item arguments.  However, in fact it steals a
# reference to the item argument!

Technically, the reference count of the value itself does not change even though you're stealing the reference.

Reply:
Tangentially related: can we add a `stolen` comment to the `PyList_SetItem`? Though the comment at the beginning of the file explicitly says this, I still think it would be useful to add it.